app/jobs/prepare_move_notifications_job.rb

```diff
@@ -41,9 +41,9 @@ def build_notification_id(subscription, type_id, topic, action_name)
 
   def should_email?(move)
     # NB: only email for:
-    #   * move.status must be Requested, Booked or Cancelled (not Proposed or Completed) moves, AND
+    #   * move.status must be Requested, Booked, In_Transit or Cancelled (not Proposed or Completed) moves, AND
```

Comment:
is the case of `In_Transit` correct? (camel and underscore case)

Reply:
the actual name is `in_transit` - I'll rename them all in this comment